Localization/StringsConvertor/input/ar.lproj/app.json

```diff
@@ -462,11 +462,23 @@
             }
         },
         "follower": {
+            "title": "follower",
             "footer": "لا يُمكِن عَرض المُتابِعين مِنَ الخوادم الأُخرى."
         },
         "following": {
+            "title": "following",
             "footer": "لا يُمكِن عَرض المُتابَعات مِنَ الخوادم الأُخرى."
         },
+        "familiarFollowers": {
+            "title": "Followers you familiar",
```

Comment:
Where is this displayed/used? I saw it in Crowdin. The correct phrase would be "Followers you follow".

Reply:
![Simulator Screen Shot - iPhone 13 Pro Max - 2022-05-17 at 23 51 58](https://user-images.githubusercontent.com/7940186/168854654-8b70c195-3f27-4c5b-99ba-c49c753a5e83.png)

Take a snapshot for it. And I'm not sure how to naming this list.